extension/app/ts/simulation/services/SimulationModeEthereumClientService.ts

```diff
@@ -581,6 +581,14 @@ const getSimulatedTokenBalances = async (ethereumClientService: EthereumClientSe
 	}))
 }
 
+const parseLogIfPossible = (ethersInterface: ethers.Interface, log: { topics: string[], data: string}) => {
+	try {
+		return ethersInterface.parseLog(log)
```

Comment:
Ethers doesn't have a `tryParse` log feature?  Might be worth filing a feature request issue on the repository or submitting a PR for it.  This seems like a generally useful thing.

Reply:
I asked ricmoo about it in discord